objects/utils.py

```diff
@@ -15,151 +15,154 @@
 if TYPE_CHECKING:
     from PIL.Image import Image
 
+
 async def flash(status: str, msg: str, template: str) -> str:
     """Flashes a success/error message on a specified template."""
-    return await render_template(f'{template}.html', flash=msg, status=status)
+    return await render_template(f"{template}.html", flash=msg, status=status)
+
 
 async def flash_with_customizations(status: str, msg: str, template: str) -> str:
     """Flashes a success/error message on a specified template. (for customisation settings)"""
-    profile_customizations = utils.has_profile_customizations(session['user_data']['id'])
+    profile_customizations = utils.has_profile_customizations(
+        session["user_data"]["id"]
+    )
     return await render_template(
-        template_name_or_list=f'{template}.html',
+        template_name_or_list=f"{template}.html",
         flash=msg,
         status=status,
-        customizations=profile_customizations
+        customizations=profile_customizations,
     )
 
+
 def get_safe_name(name: str) -> str:
     """Returns the safe version of a username."""
     # Safe name should meet few criterias.
     # - Whole name should be lower letters.
     # - Space must be replaced with _
-    return name.lower().replace(' ', '_')
+    return name.lower().replace(" ", "_")
+
 
 def convert_mode_int(mode: str) -> Optional[int]:
     """Converts mode (str) to mode (int)."""
     if mode not in _str_mode_dict:
-        print('invalid mode passed into utils.convert_mode_int?')
+        print("invalid mode passed into utils.convert_mode_int?")
         return
     return _str_mode_dict[mode]
 
-_str_mode_dict = {
-    'std': 0,
-    'taiko': 1,
-    'catch': 2,
-    'mania': 3
-}
+
+_str_mode_dict = {"std": 0, "taiko": 1, "catch": 2, "mania": 3}
+
 
 def convert_mode_str(mode: int) -> Optional[str]:
     """Converts mode (int) to mode (str)."""
     if mode not in _mode_str_dict:
-        print('invalid mode passed into utils.convert_mode_str?')
+        print("invalid mode passed into utils.convert_mode_str?")
         return
     return _mode_str_dict[mode]
 
-_mode_str_dict = {
-    0: 'std',
-    1: 'taiko',
-    2: 'catch',
-    3: 'mania'
-}
+
+_mode_str_dict = {0: "std", 1: "taiko", 2: "catch", 3: "mania"}
+
 
 async def fetch_geoloc(ip: str) -> str:
```

Comment:
This whole method should be changed to the approach that's being aimed for in bpy, via CF, maxmind and nginx

Reply:
I accdientally formatted the whole repository with double quotes, it's untouched